### DISCUSSION TITLE
Write a short bio/description for series

### DISCUSSION THREAD

**Discussion by cmgorton** [forem/forem#15474] [Icebox]
[Originally posted ](https://forem.dev/javascriptcoff1/write-a-short-bio-description-for-series-4k1i) by JavaScript➕Coffee🚀 Faye on forem.dev 

Currently, I have 2 series'; a 4 part one and a 12 part one.
As a freelance tech writer, I love the series feature as it helps me to organise my content in a way that will help people the most.

**The request**
I feel that it would be great if we could use the feature a bit more. I can share a link to my 12 part What is series, but the link is nondescript and just appears as the default DevCommunity link card.

**Goals ⚽️**
I'd like to be able to -

- Set a picture for my series
- Write a short description of my series to help people access the content.

**Stretch goal ⚽️⚽️**

- Some way to easily link between pages of a series without using full URL or separate link...

**Example of use case**
Currently, I have a stark fluctuation of view levels on the posts within the What is series, which means that readers aren't utilising, or don't know about, the series feature. Mostly my articles require no prior knowledge, however yesterday I wrote a Kubernetes article which required the reading of my Docker article first. Due to the fact that there is no way of users or writers clearly using the series feature, I had to link to my Docker article within my Kubernetes article, even though they are in the same series, which seemed strange!

I hope you can see the potential benefits that expanding the series feature would have!

Thanks for considering

Faye

**Comment by cmgorton** [comment]
Additional comments: 
https://forem.dev/michaeltharrington/comment/1pb
https://forem.dev/putsan/comment/1of
https://forem.dev/link2twenty/comment/1og

**Comment by JennieOcken** [comment]
There are a lot of requests around making our series functionality more robust.  I want to group them all together here and say we will take them as a set at some point in the future.

[Write description](https://github.com/forem/forem/discussions/15474)
[Edit](https://github.com/forem/forem/discussions/15673)
[Delete](https://github.com/forem/forem/discussions/16434)
[Embed](https://github.com/forem/forem/discussions/16805)
[Multiple Authors](https://github.com/forem/forem/discussions/15320)
[Reorder](https://github.com/forem/forem/discussions/15391)
